client/src/app/projects/pages/all-projects/all-projects.component.spec.ts

```diff
@@ -2,7 +2,7 @@ import {ComponentFixture, fakeAsync, TestBed, tick, waitForAsync} from '@angular
 import {By} from '@angular/platform-browser';
 
 import {AllProjectsComponent} from './all-projects.component';
-import {IngestService} from '../shared/services/ingest.service';
+import {IngestService} from '../../../shared/services/ingest.service';
```

Comment:
I'd love it if we could use the `@` paths instead of the relative ones. Not urgent though.

Reply:
Would need to update the angular CLI config (not a big thing) and update paths. I'll do this in a seperate PR